.github/workflows/maven-pr-builder.yml

```diff
@@ -49,9 +49,8 @@ jobs:
           ${{ runner.os }}-maven-
     # Some tests need screen access
     - name: Install xvfb
-      run: sudo apt-get install xvfb
+      run: sudo apt-get install -y xvfb
     # SonarQube scan does not work for forked repositories
     # See https://jira.sonarsource.com/browse/MMF-1371
```

Comment:
Similarly we should here elaborate that this workflow only runs for incoming pull requests and there is a separate workflow for `master` branch.

Reply:
Great!